jumpcloud-ADMU/Powershell/Start-Migration.ps1

```diff
@@ -660,13 +896,17 @@ Function Get-WindowsDrive {
   .LINK
      https://gallery.technet.microsoft.com/scriptcenter/Write-ToLog-PowerShell-999c32d0
   #>
+# Set a global parameter for debug logging
+$Script:AdminDebug = $false
+$Script:ProgressBar = $null
```

Comment:
These should be moved to Start-Migration - lets keep our code within in functions rather than these random locations

Reply:
Again, these are used in multiple functions. If we move `$Script:AdminDebug` below Write-ToLog, the value will always be $null. We populate the variable in `Start-Migration` so the parameter is initialized for `Write-ToLog` or `Update-LogTextBlock` or `Update-ProgressForm`, hence the `$script` variables needs to stay above these functions.

Reply:
We can move global variables on the very top to keep it cleaner.